.github/workflows/release-doc.yml

```diff
@@ -0,0 +1,19 @@
+name: Docs
```

Comment:
This is the "normal" one that will get triggered and deploy to S3 automatically when we publish a release, right?